internal/kubernetes/operator/config_exporter.go

```diff
@@ -395,3 +402,31 @@ func (e *ConfigExporter) exportAtlasStreamProcessing(projectName string) ([]runt
 
 	return result, nil
 }
+
+func (e *ConfigExporter) exportAtlasFederatedAuth(projectName string) ([]runtime.Object, error) {
+	result := make([]runtime.Object, 0)
+	// Gets the FederationAuthSetting
+	federatedAuthentificationSetting, err := e.dataProvider.FederationSetting(&admin.GetFederationSettingsApiParams{OrgId: e.orgID})
+	if err != nil {
+		return nil, err
+	}
+	// Does not have an IdenityProvider set then no need to geenrate
+	if federatedAuthentificationSetting.IdentityProviderStatus == nil || *federatedAuthentificationSetting.IdentityProviderStatus == "INACTIVE" {
+		return result, err
```

Comment:
Why not `nil` here as this is an error exit?
```suggestion
		return nil, err
```